### DISCUSSION TITLE
Dist-upgrade (bullseys -> bookworm) recommended?

### DISCUSSION THREAD

**Discussion by whcrg** [1000001101000/Debian_on_Buffalo#206] [General]
Hello

Is dist-upgrade from bullseye to bookworm expected to work seamlessly? Should I prepare somehow for it in advance? Or better reinstall? (Running on LS220D)

**Comment by 1000001101000** [comment]
I haven’t done a good job of creating an upgrade path for these devices. I would recommend running through the new installer rather than doing a dist-upgrade 

Some devices wouldn’t be able to boot without back porting the changes I’ve made for Bookworm. Most would mostly be fine though some would fail if the new DTB wasn’t loaded as part of that process.

for the ls220 it might have trouble picking up the MAC address since I changed how that process works, you’d need the new DTB and the “bootshim” stuff that the installer puts in place.
